operator/v1/types_ingress.go

```diff
@@ -335,8 +335,36 @@ type RouteAdmissionPolicy struct {
 	// If empty, the default is Strict.
 	// +optional
 	NamespaceOwnership NamespaceOwnershipCheck `json:"namespaceOwnership,omitempty"`
+	// wildcardPolicy describes how routes with wildcard policies should be handled
+	// for the ingress controller. WildcardPolicy works in conjunction with the
+	// wildcard policy of routes [1] claimed by the ingress controller.
+	//
+	// [1] https://github.com/openshift/api/blob/master/route/v1/types.go
+	//
+	// If WildcardPolicy is updated from WildcardsAllowed to WildcardsDisallowed,
+	// routes admitted by the ingress controller with a wildcard policy of
+	// Subdomain must be updated to a wildcard policy of None to be readmitted.
```

Comment:
What about adding a stronger warning about newly invalidated routes experiencing immediate disruption?

Reply:
@ironcladlou I updated the message to draw more attention to the caveat.